src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceNewBookingTest.kt

```diff
@@ -113,6 +117,391 @@ class OffenderResourceNewBookingTest : ResourceTest() {
           .jsonPath("bookingId").exists()
           .jsonPath("bookingNo").exists()
       }
+
+      @Test
+      @Disabled("TODO: fix this test")
```

Comment:
If I can be bothered :-) 

Yes, indeed - that is what SDI-259 is all about - but a hell of lot of work to show it doesn't work because of the interaction between our service and the NOMIS store procedure